README.md

```diff
@@ -11,42 +15,60 @@ We have different feature files for unit and integration tests. The unit tests s
+| @unit.dryrun                      | Dryrun endpoint added to Algod.                       |
+| @unit.feetest                     | Fee transaction encoding tests.                       |
+| @unit.indexer                     | Indexer REST API unit tests.                          |
+| @unit.indexer.rekey               | Rekey endpoints added to Algod and Indexer            |
+| @unit.indexer.logs                | Application logs endpoints added to Indexer.          |
+| @unit.offline                     | The first unit tests we wrote for cucumber.           |
+| @unit.rekey                       | Rekey Transaction golden tests.                       |
+| @unit.responses                   | REST Client Response serialization tests.             |
+| @unit.responses.231               | REST Client Unit Tests for Indexer 2.3.1+             |
+| @unit.responses.genesis           | REST Client Unit Tests for GetGenesis endpoint        |
+| @unit.responses.messagepack.231   | REST Client MessagePack Unit Tests for Indexer 2.3.1+ |
+| @unit.tealsign                    | Test TEAL signature utilities.                        |
+| @unit.transactions                | Transaction encoding tests.                           |
+| @unit.transactions.keyreg         | Keyreg encoding tests.                                |
+| @unit.transactions.payment        | Payment encoding tests.                               |
+
+## Integration Tests
+
+These reside in the [integration features directory](features/integration)
+
+| tag                    | description                                                                            |
+| ---------------------- | -------------------------------------------------------------------------------------- |
+| @abi                   | Test the Application Binary Interface (ABI) with atomic txn composition and execution. |
+| @algod                 | General tests against algod REST endpoints.                                            |
+| @auction               | Encode and decode bids for an auction.                                                 |
+| @application.evaldelta | Test that eval delta fields are included in algod and indexer.                         |
+| @applications.verified | Submit all types of application transactions and verify account state.                 |
+| @assets                | Submit all types of asset transactions.                                                |
+| @c2c                   | Test Contract to Contract invocations and injestion.                                   |
+| @compile               | Test the algod compile endpoint.                                                       |
+| @dryrun                | Test the algod dryrun endpoint.                                                        |
+| @dryrun.testing        | Test the testing harness that relies on dryrun endpoint. Python only.                  |
```

Comment:
A section for the tests which aren't universally implemented would be good too. I believe `@application.evaldelta` is only implemented by the Java SDK.

Reply:
👍